OneSignalSDK/onesignal/core/src/main/java/com/onesignal/session/internal/session/impl/SessionListener.kt

```diff
@@ -47,6 +47,12 @@ internal class SessionListener(
 
     override fun onSessionEnded(duration: Long) {
         val durationInSeconds = duration / 1000
+
+        // Time is invalid if below 1 second or over a day
+        if (durationInSeconds < 1L || durationInSeconds > SECONDS_IN_A_DAY) {
+            return
```

Comment:
I don't like that this can silently hide time accounting issues in the future. Thinking we should log the error.

Also think we should still send this time to OneSignal's servers, that way we know server side if we are still having issues.

Reply:
I brought it over from the [player model](https://github.com/OneSignal/OneSignal-Android-SDK/blob/8a36a3238e7cbe07ddf26494a8916bbbdcce9cc4/OneSignalSDK/onesignal/src/main/java/com/onesignal/FocusTimeController.java#L111). 

We can still send it and log an error.